mtp_api/apps/security/migrations/0030_migrate_check_description.py

```diff
@@ -0,0 +1,36 @@
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+SPLIT_CHECK_DESCRIPTION = '''
+UPDATE security_check
+SET description_migration = regexp_split_to_array(trim(description), '\\.\\s*');
+'''
+
+JOIN_CHECK_DESCRIPTION = '''
+UPDATE security_check
+SET description = array_to_string(description_migration, '. ');
+'''
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('security', '0029_remove_check_description_prefix'),
+    ]
+    operations = [
+        migrations.AddField(
+            model_name='check',
+            name='description_migration',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=100), blank=True, null=True, size=None),
```

Comment:
100 characters sounds like quite a small limit for a rule, I can see this potentially causing issues in the future